spike/CodeReaderClass.py

```diff
@@ -0,0 +1,151 @@
+class CodeParser():
+
+    #Parsing variables
+    PORT = ''
+    OPERATION = ''
+    NUMBER = ''
+
+    #Parsing libraries (or "keywords" if you wanna not sound like an arrogant prick)
+    OPERATIONS = ['+', '-', '*', '/', '=']
+    NUMBERS = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+    WHITESPACE = ' '
+    PORTS = ["THROTTLE", "TURN"]
+
+    #Initialize the CodeParser
+    def __init__(self):
+        self.PATH = "Racer.txt"
+        self.SPEED = 0
+        self.ANGLE = 0
+        self.analyzer()
+
+    #Analyze
+    def analyzer(self):
+
+        #Open racer file
+        self.racer = open(self.PATH, "r")
+
+        #Break down lines into individual parts
+        self.lexeme = ''
+
+        #Check if file is readable and then read lines
+        if self.racer.mode == "r":
+            self.code = self.racer.readlines()
```

Comment:
Reading the file in should probably happen during initialization to help ensure that the file won't be deleted before we call the analyzer